desktop_version/src/editor.cpp

```diff
@@ -1959,10 +1959,21 @@ bool editorclass::load(std::string& _path)
 
         if (pKey == "script")
         {
+            // load in the linebreak attribute
+            linebreak = "";
+            pElem->QueryStringAttribute("linebreak", &linebreak);
+            char splitvalue = linebreak[0];
+            // "" is the only value where splitvalue and linebreak are different
+            if (std::strcmp(linebreak, "") == 0)
```

Comment:
1. We prefer `SDL_strcmp()` instead of `std::strcmp()`.
2. Regardless, this check can be simplified to `linebreak[0] == '\0'`.